plugins/line-highlight/prism-line-highlight.js

```diff
@@ -39,35 +39,72 @@
 		}
 	}());
 
+	/**
+	 * Returns the top offset of the content box of the given parent and the content box of one of its children.
+	 *
+	 * @param {HTMLElement} parent
+	 * @param {HTMLElement} child
+	 */
+	function getContentBoxTopOffset(parent, child) {
+		var parentStyle = getComputedStyle(parent);
+		var childStyle = getComputedStyle(child);
+
+		/**
+		 * Returns the numeric value of the given pixel value.
+		 *
+		 * @param {string} px
+		 */
+		function pxToNumber(px) {
+			return +px.substr(0, px.length - 2);
+		}
+
+		return child.offsetTop + pxToNumber(childStyle.borderTopWidth) + pxToNumber(childStyle.paddingTop)
+			- pxToNumber(parentStyle.paddingTop);
```

Comment:
NItty style suggestion: breaking one operation onto a separate line like this makes it a bit tough to read. All on one line (so I can scan left to right) or breaking them all (so I can scan top to bottom) would be easier to read.